src/main/java/org/elasticsearch/action/admin/cluster/snapshots/delete/TransportDeleteSnapshotAction.java

```diff
@@ -64,7 +64,7 @@ protected DeleteSnapshotResponse newResponse() {
 
     @Override
     protected ClusterBlockException checkBlock(DeleteSnapshotRequest request, ClusterState state) {
-        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA, "");
+        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA_WRITE, "");
```

Comment:
.... because it remove a record created in create snapshot. So, if it's METADATA_WRITE there it should be METADATA_WRITE here.